sections/image-banner.liquid

```diff
@@ -84,7 +84,7 @@
       {%- for block in section.blocks -%}
         {%- case block.type -%}
           {%- when 'heading' -%}
-            <h2 class="banner__heading {% if block.settings.heading_size == 'medium' %}h1{% else %}h0{% endif %}" {{ block.shopify_attributes }}>
+            <h2 class="banner__heading {% if block.settings.heading_size == 'small' %}h2{% elsif 'medium' %}h1{% else %}h0{% endif %}" {{ block.shopify_attributes }}>
```

Comment:
~Here you don't want to change the value to output directly the class and remove the conditional statements ?~
I see why in slack 😅  We don't want to introduce a breaking change. 👍